docs/content/quickstart/_index.md

```diff
@@ -128,6 +129,27 @@ Upgraded to World 2.0
 execution completed successfully!
 ```
 
+### Upgrade using digest
+
+When working in production environment, we highly recommend you to reference the bundle using its digest instead of tag. We used tags in our docs for simplicity, but tags can be overwritten which results in unexpected outcomes from upgrading an bundle.
+To get deterministic and repeatable deployments, use digests instead of tags. it allows to ensure that you deploy the exact expected bundle.
```

Comment:
```suggestion
For deterministic and repeatable deployments, use digests instead of tags to ensure that you deploy the exactly what you intended.
```